src/main/java/com/selfxdsd/selfpm/Webhooks.java

```diff
@@ -143,10 +145,20 @@ public Webhooks(final Self selfCore, final SelfTodos selfTodos) {
                     Provider.Names.GITHUB
                 );
                 if (project == null) {
-                    LOG.debug(
-                        "Project " + fullName + " not found either. No Content."
+                    LOG.debug("Project not found, trying changes.repository");
+                    final String fullNameRenamed = this
+                        .getFullNameFromChanges(jsonPayload);
+                    project = this.selfCore.projects().getProjectById(
+                        fullNameRenamed,
+                        Provider.Names.GITHUB
                     );
-                    return ResponseEntity.noContent().build();
+                    if(project == null) {
+                        LOG.debug(
+                            "Project " + fullName + " not found either."
+                                + " No Content."
+                        );
+                        return ResponseEntity.noContent().build();
+                    }
```

Comment:
@criske it should be the other way: first look for ``changes.repository`` and if the full name is not found there, take it from ``repository.full_name``. 

Also please write a TODO for adding a unit test or two for this change :D